tools/check_format.py

```diff
@@ -172,14 +172,64 @@ def fixJavaProtoOptions(file_path):
   return []
 
 
-def checkJavaProtoOptions(file_path):
+# Add 'option java_outclass_name = FooBarProto' for foo_bar.proto
+def fixJavaOuterClassnameProtoOption(file_path):
+  file_name = os.path.basename(file_path)[:-len(".proto")]
+  if "-" in file_name or "." in file_name or not file_name.islower():
+    return ["Unable to decide java_outer_classname for proto file: %s" % file_path]
+
+  for line in fileinput.FileInput(file_path):
+    if "option java_outer_classname = " in line:
+      return []
+
+  to_add = "option java_outer_classname = \"" + ''.join(
+      x.title() for x in file_name.split('_')) + "Proto\";\n"
+
+  for line in fileinput.FileInput(file_path, inplace=True):
+    if line.startswith("package "):
+      line = line.replace(line, line + to_add)
+    sys.stdout.write(line)
+
+  return []
+
+
+def fixJavaMultipleFilesProtoOption(file_path):
+  for line in fileinput.FileInput(file_path):
+    if "option java_multiple_files = " in line:
```

Comment:
Same here, should this check for true?

Reply:
Same here, allow purposely setting it to false for some particular proto file.